resources/metadata/sonar.yaml

```diff
@@ -248,4 +248,4 @@ spec:
               - name: sonar
   containers:
     - name: sonar
-      image: node:lts-stretch
+      image: sonarsource/sonar-scanner-cli:4.5
```

Comment:
Not sure about this change. Why did we not use the `sonar-scanner-cli` image before? Why use it now?

Reply:
see https://github.com/SAP/jenkins-library/issues/1072